src/Controls/tests/UITests/Tests/Issues/Issue20631.cs

```diff
@@ -0,0 +1,74 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues;
+
+public class Issue20631 : _IssuesUITest
+{
+	public Issue20631(TestDevice device) : base(device) { }
+
+	public override string Issue => "Editor does not scroll when cursor goes behind keyboard";
+
+	[Test]
+	[Category(UITestCategories.Entry)]
+	public void TestEditorKeyboardScrolling ()
```

Comment:
The test is failing on iOS. Could you take a look?

`Expected string length 795 but was 796. Strings differ at index 710.`

`at Microsoft.Maui.AppiumTests.Issues.Issue20631.TestEditorKeyboardScrolling() in /Users/builder/azdo/_work/2/s/src/Controls/tests/UITests/Tests/Issues/Issue20631.cs:line 58`